### ISSUE TITLE
Causes title bar display error on windows 11 dev build 22557

### ISSUE THREAD

**Issue by anonymousgregory** [Tatsu-syo/noMeiryoUI#39]
when changing the title bar font in noMeiryoUI, title bars get transparent and show no characters at all in many title bar. This might be due to the i[mplantation of Mica in the title bar of more dialogs.](https://blogs.windows.com/windows-insider/2022/02/16/announcing-windows-11-insider-preview-build-22557/#:~:text=Insiders%20will%20notice%20the%20Mica%20material%20in%20more%20title%20bar%20surfaces.%20For%20example%2C%20the%20Run%20dialog%20now%20utilizes%20Mica.)

**Comment by Tatsu-syo** [comment]
What?
When autumn comes in America, What is happened? I don't know.
If appearance becomes more beautiful, it's a happy for many people in many countries

Except in Japan!!
Since 10 years ago, Japanese user interface designers work is too bad.
Especially in fonts. In Windows 8, Meiryo UI is completely broken, and most ugly in Windows history.
I was very angry, and made No!! Meiryo UI.
In Windows 10, Yu Gothic UI is bit better, but still poor. In Windows 11 too.
Now, I think Japanese staffs do no work. They are wage thieves.
New notepad and media player looks is good but the text looks poor in Japan.
This is Japan's reality.
I feel very sad, angry, downhearted..

Worst scenario:
If the system font settings have no effects in future Windows, This project become maintenance mode.
And after Windows 10 support ends, this project is end.

I can only see what happens.

**Comment by MyAlfisti** [comment]
This bug also occurs in 22563.1.
I have removed this application from my storage, cursing the fact that I am in a 2-byte character area.

**Comment by Tatsu-syo** [comment]
I also live in 2-byte character area too.
Multi bytes character draw routine is different from single byte character draw routine?

Should I try the insider program? Does insider program requires another license?

In addition I'm working on another software using .NET Framework and WinForms(closed source).
It requires modernize for my use, for study for my job.
I'm planning to try this.

This issue occurred  in under developing Windows.
I will see how it goes for a while.

**Comment by Tatsu-syo** [comment]
On Windows 11 dev build 22581, No!! Meiryo UI is worked and can change system fonts.
It's a transitional glitch.
If glitch remains, some countries where system font is not Segoe UI, system widgets can't display correctly.

![Dev22581](https://user-images.githubusercontent.com/24761091/160613764-89fece37-33b8-453f-ba88-8c0b18f53dbf.png)

**Comment by zys52712** [comment]
> On Windows 11 dev build 22581, No!! Meiryo UI is worked and can change system fonts. It's a transitional glitch. If glitch remains, some countries where system font is not Segoe UI, system widgets can't display correctly.
> 
> ![Dev22581](https://user-images.githubusercontent.com/24761091/160613764-89fece37-33b8-453f-ba88-8c0b18f53dbf.png)

Can you try it with transparency enabled?
It's located under Windows Settings>Personalization>Colors>Transparency Effects

**Comment by Tatsu-syo** [comment]
I get stuck.
I can't check this issue.
Now, I use insider preview in evaluation version.
But, it's weird.
@zys52712 If you change single byte character font like Segoe UI and this issue occur?

Now, I can only see how it goes for a while.

**Comment by zys52712** [comment]
> I get stuck. I can't check this issue. Now, I use insider preview in evaluation version. But, it's weird. @zys52712 If you change single byte character font like Segoe UI and this issue occur?
> 
> Now, I can only see how it goes for a while.

Single byte ones do work it seems.  Arial, Segoe UI etc.

**Comment by Tatsu-syo** [comment]
Hmm, Multi byte charactor issue?

**Comment by eliaszon** [comment]
I got the same issue on Windows dev build 22598. It really sucks. Almost the system interfaces are transparent and can't open up even the system restore interface. I have been working on this HUGE problem for 6 hours and finally work it out. 
HERE is the solution:
enter the safe mode and restore your system. one caveate: you have to set a restore point before!!!

**Comment by eliaszon** [comment]
imp, it's super IMPORTANT to notice your users of the risk and let them prepare in advance. Because it's really such a big BUG. Once this happens, it almost can't fix it and the user have to reinstall the system and it will cause data problem. I strongly suggest that the authors pop up a pop-up window in the software interface to warn of the risks.  Thanks!

**Comment by Tatsu-syo** [comment]
What? It's very serious issue.
If this problem occurs release preview channel, I will add warning on Windows 11 and move this project maintenance mode.
Thanks.

**Comment by Tatsu-syo** [comment]
I tested Windows 11 dev build 25120.1010 and issue not occurd.
Because of different build?
Because of test environment? I tested Windows 11 on HYPER-V.
The biggest differences ares video card and driver.
@eliaszon What card do you use?

For those who are watching, should I use real machine for test Windows Insider preview?
I see how it goes for a while.

**Comment by anonymousgregory** [comment]
> I tested Windows 11 dev build 25120.1010 and issue not occurd. Because of different build? Because of test environment? I tested Windows 11 on HYPER-V. The biggest differences ares video card and driver. @eliaszon What card do you use?
> 
> For those who are watching, should I use real machine for test Windows Insider preview? I see how it goes for a while.

![image](https://user-images.githubusercontent.com/28359613/169697668-22b4abc3-ab46-49f2-9f9f-067667821291.png)
I'm on latest build and the app works fine except the title bar issue. Didn't seem to have what @eliaszon mentioned here

**Comment by eliaszon** [comment]
> I tested Windows 11 dev build 25120.1010 and issue not occurd.
> Because of different build?
> Because of test environment? I tested Windows 11 on HYPER-V.
> The biggest differences ares video card and driver.
> @eliaszon What card do you use?
> 
> For those who are watching, should I use real machine for test Windows Insider preview?
> I see how it goes for a while.

It’s the title bar issue. Maybe I patched the system for 3-party theme. Maybe? If U don't change the title bar font, everything will be fine.

**Comment by eliaszon** [comment]
> > I tested Windows 11 dev build 25120.1010 and issue not occurd. Because of different build? Because of test environment? I tested Windows 11 on HYPER-V. The biggest differences ares video card and driver. @eliaszon What card do you use?
> > 
> > For those who are watching, should I use real machine for test Windows Insider preview? I see how it goes for a while.
> 
> ![image](https://user-images.githubusercontent.com/28359613/169697668-22b4abc3-ab46-49f2-9f9f-067667821291.png)
> I'm on latest build and the app works fine except the title bar issue. Didn't seem to have what @eliaszon mentioned here
> 
> 

Agree.It's the title bar issue. If I don't change the title bar font, everything will be ok. If I changed it,the title bar will be transparent.Then u can't move the software window and system window.
